my/lessons/advanced/concurrency.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: Keserempakan(Concurrency)
```

Comment:
I'm concerned these `(...)` sections in the titles will cause issues with the UI and CSS.  Can we remove them @alkaha?  

Perhaps we should move all instances of `something (english)` to the Glossary.  Thought?